lib/plugin.js

```diff
@@ -47,80 +49,95 @@ class I18nModularPlugin {
     const keysRoot = rebase(compiler.context, relativeKeysRoot);
     const dictionaryRegex = getDictionaryRegex(rebase(compiler.context, dictionaryPattern));
 
+    // Stops watching dictionary files to avoid rebuild loops in watch mode
```

Comment:
maybe migrate some of those comments over to functions with appropriate names?

Reply:
instance methods perhaps, but still would need too many arguments

Reply:
Would rather have a function with many arguments vs comments that get outdated

Reply:
treat it as a step towards JSDoc!

Reply:
JSno 😔